recipe/__init__.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+from __future__ import print_function, absolute_import
+
+__version__ = "0.11.7"
```

Comment:
That said I can pick something else it looks like. For example, `__version__ = "$VERSION"` or similar as it replaces all lines with `__version__` in them.